### PR TITLE
Check name similarity when adding exercises

### PR DESCRIPTION
**Pull request by lydiaxing**
### Proposed Changes
Addressed [issue #551](https://github.com/wger-project/wger/issues/551) by adding a cleaning function to the exercise submission form which checks for name similarity to existing exercises via the levenshtein algorithm
image

### Please check that the PR fulfills these requirements

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] New python code has been linted with with flake8 (``flake8 --config .github/linters/.flake8``)
and isort (``isort``) 
- [x] Added yourself to AUTHORS.rst

### Other questions

* Does this PR introduce a breaking change such as a database migration? (i.e.
what changes might users need to make in their running application due to
this PR?)
No

Thank you!

### CODE REVIEW

File: wger/utils/language.py [hunk 1]
```diff
@@ -99,7 +100,7 @@ def load_ingredient_languages(request):
     """
 
     language = load_language()
-    languages = load_item_languages(LanguageConfig.SHOW_ITEM_INGREDIENTS)
+    languages = load_item_languages('2')
```
Comment (rolandgeider) [inline wger/utils/language.py:103]: Mhh, having magic numbers (or strings) is really not a good idea. We could move SHOW_ITEM_INGREDIENTS to utils/constants.py, that would probably break the circular import. Having the import inside the function is OK if it's necessary, but better write a short comment why.

The whole LanguageConfig for ingredients will ideally go away when the Open Food Facts issue is done and the DB has more products (the config is used so that if e.g. a language has only a handful of exercises, you can configure the application to also show those users exercises in english and the application is more useful)

**Comment by rolandgeider** [comment]
Thanks for the PR! I want to do some more testing before I run this on the production database (on a test run I noticed how there are almost-but-not-quite duplicates of products) and also need to think about how to distribute the fixtures after this

File: wger/exercises/views/exercises.py [hunk 2]
```diff
@@ -157,6 +160,59 @@ def view(request, id, slug=None):
...
+    class Media:
+        js = (settings.STATIC_URL + 'yarn/tinymce/tinymce.min.js',)
+
+    def clean_name_original(self):
+        """
+        Throws a validation error if the submitted name is too similar to an existing
+        exercise's name
+        """
+        name_original = self.cleaned_data['name_original']
+        languages = load_item_languages(LanguageConfig.SHOW_ITEM_EXERCISES)
```
Comment (rolandgeider) [inline wger/exercises/views/exercises.py:202]: You don't need to load the languages from config, here it's enough if you just filter by the one fom the current exercise. The LanguageConfig is meant so that if e.g. Russian has only a handful of exercises and ingredients, you can configure the application to also show exercises in English so that it's a bit more useful. In our case here, it's perfectly OK that there exist different exercises with the same name but in different languages

**Review by rolandgeider — CHANGES_REQUESTED** [review]

**Review by rolandgeider — APPROVED** [review]

**Comment by rolandgeider** [comment]
Merged, thanks!!
